### DISCUSSION TITLE
blog/adding-i18n-for-a-docusaurus-site/

### DISCUSSION THREAD

**Discussion by giscus** [sqybi/SQYBI.com#5] [Announcements]
# blog/adding-i18n-for-a-docusaurus-site/

在发布了建站的第一篇文章后，本想趁着三分钟热度抓紧多更新几篇的。

https://sqybi.com/blog/adding-i18n-for-a-docusaurus-site/

**Comment by Dandi007** [comment]
最近在用LLM学习CMake（之前更多是run通就足够了，不咋去深入了解它的语法细节），效果确实很让人惊艳（唯一的不足就是会担心它胡编乱造了hhhhh
LLM刚出的时候偷懒没有深入去学习使用，有一些焦虑在里面的，觉得人不学习的话就会落后时代而被淘汰掉，或者更糟——用轻视的态度掩盖内心的恐慌，拒绝接受新鲜的事物。

**Reply by sqybi** [reply]
@Dandi007 其实我反而觉得大模型的使用本身没有什么可深入学习的（除非你真的想研究这个领域）。如何更好地利用大模型，和你是不是一个好的产品经理可能反而更相关 :P

**Comment by ktcjxtq** [comment]
怎么还要注册github才能评论[○･｀Д´･ ○]

**Comment by sqybi** [comment]
补充一下，今天改了第一篇文章的几个 typo 之后，突然发现虽然原文打错了，但是英文翻译几乎全都翻译对了……
只能说 LLM 比我强多了，甘拜下风，下次让他顺便也帮我检查一下错字吧 :laughing:

**Comment by Oreoxmt** [comment]
真的是 18 个字母耶 🎉

**Reply by sqybi** [reply]
没骗你吧！
